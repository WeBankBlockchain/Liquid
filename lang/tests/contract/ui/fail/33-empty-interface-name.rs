use liquid_lang as liquid;

#[liquid::interface(name = "")]
mod foo {
    extern "liquid" {
        fn bar();
    }
}

fn main() {}