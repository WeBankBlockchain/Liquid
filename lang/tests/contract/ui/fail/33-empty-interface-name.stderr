error: invalid interface name: ``
 --> $DIR/33-empty-interface-name.rs:3:28
  |
3 | #[liquid::interface(name = "")]
  |                            ^^
